ibm/mas_devops/roles/suite_db2_setup_for_manage/tasks/main.yml

```diff
@@ -77,13 +77,30 @@
 - name: Run setupdb to prepare Db2 for use with Maximo TPAE
   shell: oc exec -n {{db2_namespace}} {{db2_pod_name}} -- su -lc '/tmp/setupdb.sh | tee /tmp/setupdb.log' db2inst1
   register: prepare_cmds_status
+  retries: 10
+  delay: 30 # seconds
+  until: prepare_cmds_status.rc == 0
+
+- name: "Print setupdb.sh return code"
+  when: prepare_cmds_status.rc is defined
+  debug:
+    msg:
+      - "SetupDB return code (rc) ........................ {{ prepare_cmds_status.rc }}"
 
 - name: "Database setup debug information"
   debug:
     msg:
       - "Result (stdout) ........................ {{ prepare_cmds_status.stdout }}"
       - "result (stderr) ........................ {{ prepare_cmds_status.stderr }}"
 
+- name: Assert that manage tablespace or bufferpools are not created
```

Comment:
Shouldn't this say "Assert that manage tablespace or bufferpools are created" rather than "Assert that manage tablespace or bufferpools are **not** created"

Reply:
Yes, you're right!